api.swagger.yaml

```diff
@@ -12469,6 +12495,51 @@ components:
                     type: string
                     format: uuid
                     example: '01234567-89ab-cdef-1234-56789abcdef0'
+            free_skills:
+              description: Free Skills
+              type: array
+              items:
+                type: object
+                properties:
+                  uuid:
+                    type: string
+                    format: uuid
+                    example: '01234567-89ab-cdef-1234-56789abcdef0'
+            personal_interests:
+              description: Personal Interests
+              type: array
+              items:
+                type: object
+                properties:
+                  uuid:
+                    type: string
+                    format: uuid
+                    example: '01234567-89ab-cdef-1234-56789abcdef0'
+            social_media:
```

Comment:
social_media should be in user-schema-common (rule is if its same in GET & POST should be in common)